backend/fms_core/serializers.py

```diff
@@ -221,13 +223,13 @@ class Meta:
 
 
 class TaxonSerializer(serializers.ModelSerializer):
-    editable = serializers.SerializerMethodField()
+    editable = serializers.BooleanField(read_only=True)
     class Meta:
         model = Taxon
         fields = "__all__"
         extra_fields = ("editable")
```

Comment:
This extra_field is not present in the response... you likely need to add this field to the fields instead of extra_fields ...